microceph/ceph/osd.go

```diff
@@ -1069,3 +1069,50 @@ func killOSD(osd int64) error {
 	}
 	return nil
 }
+
+func SetReplicationFactor(pools []string, size int64) error {
+	ssize := fmt.Sprintf("%d", size)
+    _, err := processExec.RunCommand("ceph", "config", "set", "global",
+                                     "osd_pool_default_size", ssize)
+    if err != nil {
+        return fmt.Errorf("failed to set pool size default: %w", err)
+    }
+
```

Comment:
Is the default pool size set operation being performed always ? (I couldn't find any check that triggers this only if an empty string is provided.)